### ISSUE TITLE
lambda for the losses

### ISSUE THREAD

**Issue by ahyunSeo** [cvlab-yonsei/projects#2]
Hello,

I have a question about the experiments on the Table 3: Average PCK comparison of different loss functions. 

In the paper you stated that you fixed the parameters as (λmask = 3, λflow = 16, λsmooth = 0.5).

How did you set the parameters in your ablation studies?

For example, If you train only with the two losses (Lmask, L_flow), did you set the λmask and λflow as (3, 16) or did you find another combination?

Best regards,

Ahyun

**Comment by Shape-Kim** [comment]
Thank you for question.

The parameters for losses are fixed to all experiments.
For example, we set the λmask and λflow as (3, 16) when training the network only with two losses (L_mask, L_flow).

Thank you.
